crates/server/src/server/server_future.rs

```diff
@@ -939,31 +1045,39 @@ mod tests {
     use super::*;
     use crate::authority::Catalog;
     use futures_util::future;
-    use std::net::{Ipv4Addr, SocketAddr, UdpSocket};
+    use std::net::SocketAddr;
+    use tokio::net::{TcpListener, UdpSocket};
+    use tokio::time::timeout;
 
-    #[test]
```

Comment:
Why is this test no longer valuable?

Reply:
It is, I just renamed it to `abort` to distinguish it from the test for `graceful_shutdown`.